src/main/java/seedu/address/logic/commands/DeletePropertyCommand.java

```diff
@@ -55,10 +53,17 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_INPUT);
         }
 
-        Person personWithoutProperty = new Person(personToDeleteProperty.getName(),
-                personToDeleteProperty.getPhone(), personToDeleteProperty.getEmail(),
-                personToDeleteProperty.getTags(), personToDeleteProperty.getAppointment(), new Property(""));
-        model.setPerson(personToDeleteProperty, personWithoutProperty);
+        Person personWithoutProperty;
```

Comment:
Similar to DeleteAppointmentCommand. Perhaps making common fields can decrease the code repetition.